c++/CMakeLists.txt

```diff
@@ -82,11 +82,40 @@ else()
 endif()
 
 # Source =======================================================================
-
+include(CapnProtoMacros)
 add_subdirectory(src)
 
 # Install ======================================================================
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake"
+  VERSION ${VERSION}
+  COMPATIBILITY AnyNewerVersion
```

Comment:
If this is the Cap'n Proto API we're talking about, I think it's safe enough to assume that newer versions will be source-compatible, but not binary-compatible. (It's true that we sometimes make breaking changes to the API but only in obscure parts. Anything widely-used will be gracefully deprecated.)